src/Lib/HTMLBuilder.php

```diff
@@ -139,14 +157,18 @@ public static function createCheckboxList(array $checkbox_list, array|null $chec
 
         $fragment = "";
         foreach ($checkbox_list as $checkbox) {
-            $id_replaced = str_replace(search: "%id%", replace: $checkbox->id, subject: $checkbox_part);
-            $name_replaced = str_replace(search: "%tag_name%", replace: $checkbox->name, subject: $id_replaced);
+            $ui_material_list = array(
+                new UIMaterial(slot: "id", replacement: $checkbox->id),
+                new UIMaterial(slot: "tag_name", replacement: $checkbox->name),
+            );
```

Comment:
```suggestion
            $ui_material_list = [
                new UIMaterial(slot: "id", replacement: $checkbox->id),
                new UIMaterial(slot: "tag_name", replacement: $checkbox->name),
            ];
```
arrayは短縮構文を使ったほうが他の言語の気泡と近いのでおすすめです。